content/docs/command-reference/spec.json

```diff
@@ -0,0 +1,734 @@
+{
+  "apply": {
+    "args": {
+      "args": [
+        {
+          "decl": "model",
+          "help": "Path to model object [required]"
+        },
+        {
+          "decl": "data",
+          "help": "Path to data object [required]"
+        }
+      ],
+      "impls": null,
+      "impl_metavar": null,
+      "subcommands": null
+    },
+    "options": [
+      {
+        "decl": "-p, --project TEXT",
+        "help": "Path to MLEM project [default: (none)]"
+      },
+      {
+        "decl": "--rev TEXT",
+        "help": "Repo revision to use [default: (none)]"
+      },
+      {
+        "decl": "-o, --output TEXT",
+        "help": "Where to store the outputs."
+      },
+      {
+        "decl": "-m, --method TEXT",
+        "help": "Which model method is to be applied [default: predict]"
+      },
+      {
+        "decl": "--data-project, --dr TEXT",
+        "help": "Project with data"
+      },
+      {
+        "decl": "--data-rev TEXT",
+        "help": "Revision of data"
+      },
+      {
+        "decl": "-i, --import",
+        "help": "Try to import data on-the-fly"
+      },
+      {
+        "decl": "--import-type, --it TEXT",
+        "help": "Specify how to read data file for import. Available types: ['pandas', 'pickle']"
+      },
+      {
+        "decl": "-b, --batch_size INTEGER",
+        "help": "Batch size for reading data in batches."
+      },
+      {
+        "decl": "--index / --no-index",
+        "help": "Whether to index output in .mlem directory"
+      },
+      {
+        "decl": "-e, --external",
+        "help": "Save result not in .mlem, but directly in project"
+      },
+      {
+        "decl": "--json",
+        "help": "Output as json"
+      },
+      {
+        "decl": "--help",
+        "help": "Show this message and exit."
+      }
+    ],
+    "usage": "Usage: mlem apply [options] model data",
+    "doc": "Apply a model to data. The result will be saved as a MLEM object to `output` if\nprovided. Otherwise, it will be printed to `stdout`."
```

Comment:
Q: When is the `\n` inserted? The [original string](https://github.com/iterative/mlem/blob/4666de893967cdde41bf5ff605acf51a84bc9b68/mlem/cli/apply.py#L68) doesn't have it.

Reply:
I guess that is wrong commit. [This one](https://github.com/iterative/mlem/blob/1f7bc2952bebaf0fd9dc7269d620193d7c1f44a7/mlem/cli/apply.py) is more up-to-date

Reply:
I'm seeing it in the PR changes: https://github.com/iterative/mlem.ai/pull/172/files#diff-5bc25890d642b6f5b21be6b3e1ea5fb710777e00c7c51c86b6aaebf71a971e1d maybe spec.json needs to be generated again (or removed) for the PR?

> Your link is to mlem/cli/apply.py  Confused 😋

Reply:
yours too :) 
Yeah I will remove spec from PR now